hugo/content/_index.html

```diff
@@ -47,9 +47,10 @@ <h2>Deploy on-premises</h2>
 </section>
 <section>
     <div class="text-content">
+        <hr>
```

Comment:
Personal preference: I think it would look nicer if the hr had the same length as the previous section. Something like this
![grafik](https://github.com/TypeFox/oct-website/assets/34068281/869d4260-a796-483d-8448-f410965a64f0)

Reply:
That's not so easy because the hr doesn't work well with the flex container. I'd keep it as is for now.